src/main/java/frc/robot/subsystems/vision/Vision.java

```diff
@@ -0,0 +1,47 @@
+package frc.robot.subsystems.vision;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Robot;
+import org.littletonrobotics.junction.Logger;
+import org.photonvision.PhotonCamera;
+
+public class Vision extends SubsystemBase {
+
+    private static Vision INSTANCE = null;
+
+    private final VisionModule[] modules;
+    private final Result[] results;
+
+    private Vision(VisionModule... modules) {
+        this.modules = modules;
+        results = new Result[modules.length];
+    }
+
+    public static Vision getInstance() {
+        if (INSTANCE == null) {
+            if (Robot.isReal()) {
+                INSTANCE = new Vision(
+                        VisionModule.photonVisionIO("LeftPie", 0),
+                        VisionModule.photonVisionIO("RightPie", 1)
```

Comment:
"Pi", not "Pie" 🙃.  
Also, it's better to write "Left Pi" and "Right Pi" -- this is a string, no need to join words.